markdown_anki_decks/cli.py

```diff
@@ -48,6 +48,11 @@ def convertMarkdown(
         "--prefix",
         help="Can be used to make your markdown decks part of a single subdeck. Anki uses `::` to indicate sub decks. `markdown-decks::` could be used to make all generated decks part of a single root deck `markdown-decks`",
     ),
+    filetype: str = typer.Option(
+        "md",
+        "--ft",
+        help="Can be use to specify the use of other filetypes. Specify just the fine extension, e.g., 'txt'",
```

Comment:
Change the help text to something like.

"This option lets you specify which file extensions markdown-anki-decks will use when searching for files to convert into Anki decks. By default, it uses 'md' as the only file extension."